src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/DisplayLine.kt

```diff
@@ -53,12 +57,17 @@ abstract class DisplayLine {
 
     init {
         MouseListener.registerClickListener { x, y, button, pressed ->
-            if (x in cachedX..cachedX + cachedWidth && y in cachedY..cachedY + cachedHeight)
+            if (
+                shouldRender &&
+                x in cachedX..cachedX + cachedWidth &&
+                y in cachedY..cachedY + cachedHeight
+            )
                 onClicked?.trigger(arrayOf(x, y, button, pressed))
```

Comment:
I like to put braces around `if` statements with a multiline condition, even if they're only one line